src/scene/vcMedia.cpp

```diff
@@ -148,9 +148,9 @@ void vcMedia::AddToScene(vcState *pProgramState, vcRenderData *pRenderData)
     udInt2 imageSize = {};
     if (m_pImageData != nullptr)
```

Comment:
In general, this file looks like it should have use the texture cache instead (outside the scope of this PR)

Reply:
Ok. Is adapting `vcMedia` to use the texture cache within the scope of this PR? As I write this I would imagine it should probably be it's own PR.

Reply:
Should I roll this back?